### PR TITLE
feat(TYTA-39): add team controller

### PR DESCRIPTION
**Pull request by kilianpichard**

### CODE REVIEW

File: src/controllers/teamCtrl.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+  )
+    .then((team) => {
+      res
+        .status(200)
+        .json({ message: 'The user has been removed from the team!', team });
+    })
+    .catch((error) =>
+      res.status(401).json({ message: 'Invalid Request!', error })
+    );
```
Comment (remirubis) [inline src/controllers/teamCtrl.js]: Some changes to be made in connection with the changes requested here https://github.com/TimeYourTask/Tyta-cors/pull/7

**Review by remirubis — COMMENTED** [review]
Some changes to be made in connection with the changes requested [here](https://github.com/TimeYourTask/Tyta-cors/pull/7)

**Review by remirubis — APPROVED** [review]
LGTM! 🚀

**Review by BrystoQ — APPROVED** [review]
NTM (Nice To Me)
Just wait merge other request before

**Pull request merged**